javascript-functions/main.js

```diff
@@ -0,0 +1,41 @@
+function convertMinutesToSeconds(minutes) {
+  var convert = minutes * 60;
```

Comment:
I would suggest having a variable name such as `seconds` instead of `convert` as I think it would bring more clarity to what that variable is holding.